### DISCUSSION TITLE
'\t' inserted in place of indentation

### DISCUSSION THREAD

**Discussion by yeahitsaustin** [codefori/vscode-ibmi#1930] [Q&A] [answered]
Good afternoon,

We've had some users report unwanted characters '\t' and '\z' getting inserted into code when saving source members. We believe it has to do with goofy indentation originally done in PDM. For example this code in QSQLDDLSRC:

![image](https://github.com/codefori/vscode-ibmi/assets/151177600/98905123-fd64-4692-a3e3-3ef4f635ce58)
 
Upon saving and reopening in VS Code, '\t' appears where there was indentation:

![image](https://github.com/codefori/vscode-ibmi/assets/151177600/c662afe0-a774-4b5e-aa67-3b1af0fcbd32)

Any insight into what might be causing this or how to avoid it? Unfortunately, stripping out the unwanted characters manually isn't the preferred option for us because that would update the source dates.

Thank you,

**Comment by worksofliam** [comment, marked as answer]
Hi @yeahitsaustin - there is actually a PR open for this right now so we should have a fix out in a few days.

In the mean time, consider changing the VS Code defaults to use spaces when inserting a tab.

**Reply by yeahitsaustin** [reply]
Awesome, thanks for the reply.
